.github/workflows/tests.yml

```diff
@@ -1,5 +1,8 @@
 name: Run tests
 
+env:
```

Comment:
While Playwright works locally, I was getting errors when it ran in the CI. I've spent a few hours on this already , and not worth it trying to fix it right now, so I disabled the Playwright tests in the CI.

What baffles me, tho, is that I don't know why this is happening. But it looks like that, when we run `tox` in the CI, then the tests are running in an asyncio loop. This trips up Playwright, because Playwright contains logic that raises if you run it synchronously inside an async loop.

However, the synchronous test execution is required, because I used `[StaticLiveServerTestCase](https://docs.djangoproject.com/en/5.1/ref/contrib/staticfiles/#django.contrib.staticfiles.testing.StaticLiveServerTestCase)` to be able to spin up a Django server (that also serves static files) during the tests. And Django doesn't provide an async variant of `StaticLiveServerTestCase`.

So IMO the options are to either to:
1. Re-implement `StaticLiveServerTestCase` to support async, and rewrite tests to be async.
2. Move test server mgmt out of tests. And instead spin up a dev server before the tests, and then shut it down after the tests are done.

Option 2. sounds more sensible, but can be tackled as a standalone issue.

---

One of the potential suspects of this CI issue was that maybe the CI was using different versions of dependencies. So I added `requirements-test.in` to pin the version of `tox` used in CI.

Reply:
I'm afraid I get the same errors when running tox locally. Did you try running it after deleting the .tox dir?

```
E               playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.
E               Please use the Async API instead.

.tox/py311/lib/python3.11/site-packages/playwright/sync_api/_context_manager.py:47: Error
```

Reply:
Actually, I'm getting this when just running pytest as well (needed `pip install -e .` to get it working), so this likely isn't a tox problem either.